Task1.5/Script.js

```diff
@@ -0,0 +1,20 @@
+﻿let arr = [1,2,3,2,3,2,2,3,3,1,5,3,2]
+let arr2 = [1,2,3,4,4,1,2,3,3,6]
+
+function findAlone(arr) {
+    for (let i = 0; i < arr.length; i++)
+    {
+        let count = 0
+        for (let j = 0; j < arr.length; j++){
+            if (arr[j] === arr[i]) count++
```

Comment:
Возвращаясь к той же читабельности кода, давай в будущем всегда обрамлять тело if скобками. Иначе может привести к ряду ошибок и путаниц. Кроме того, при использовании различных code formatters, тело может улететь на другую строчку, что равносильно пустому return.
Переходим на это:
```suggestion
            if (arr[j] === arr[i]) {
              count++;
             }
```